<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GIS Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabContainer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="gisCalc">
     <attribute name="title">
      <string>Gis Calculator</string>
     </attribute>
     <widget class="QGroupBox" name="centroidGroup">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>20</y>
        <width>361</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="centroidCalcLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Calculate Origin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="originClearButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="originCalculateButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>470</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
      <widget class="QLabel" name="originResultsLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Origin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QRadioButton" name="ddButtonCalcOrigin_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Decimal Degrees</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="llButtonCalcOrigin_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Lat/Lon</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Coordinate System</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="westXEntry">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>108</y>
         <width>103</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>East X</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="eastXEntry">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>108</y>
         <width>103</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>West X</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="northYEntry">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>60</y>
         <width>103</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>North Y</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="southYEntry">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>160</y>
         <width>103</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>South Y</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>88</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>West X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>88</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>East X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>40</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>North Y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>140</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>South Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="originOutputBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>376</y>
         <width>321</width>
         <height>29</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>centroidCalcLabel</zorder>
      <zorder>originClearButton</zorder>
      <zorder>originCalculateButton</zorder>
      <zorder>originResultsLabel</zorder>
      <zorder>ddButtonCalcOrigin_2</zorder>
      <zorder>llButtonCalcOrigin_4</zorder>
      <zorder>label</zorder>
      <zorder>westXEntry</zorder>
      <zorder>eastXEntry</zorder>
      <zorder>northYEntry</zorder>
      <zorder>southYEntry</zorder>
      <zorder>label_3</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>originOutputBox</zorder>
      <zorder>coordinatesResultsLabel</zorder>
     </widget>
     <widget class="QGroupBox" name="originGroup">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>361</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="centroidCalcLabel_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Convert Coordinates</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="convCoordClear">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="convCoordCalc">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>470</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
      <widget class="QComboBox" name="conversionCombo">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>291</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Lat/Lon to Decimal Degrees</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Decimal Degrees to Lat/Lon</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="westXEntry_2">
       <property name="geometry">
        <rect>
         <x>241</x>
         <y>108</y>
         <width>103</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>West X</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="eastXEntry_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>108</y>
         <width>103</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>East X</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>88</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>88</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>376</y>
         <width>321</width>
         <height>29</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="coordinatesResultsLabel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>368</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Converted Coordinates</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>originGroup</zorder>
     <zorder>centroidGroup</zorder>
     <zorder>coordinatesResultsLabel</zorder>
    </widget>
    <widget class="QWidget" name="catalogTiff">
     <attribute name="title">
      <string>Catalog Tiffs</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Geotiff Directory: </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="overwriteMode">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="catalogTiffBrowseButton">
      <property name="geometry">
       <rect>
        <x>652</x>
        <y>26</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="tiffCatalogOutputLabel">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>78</y>
        <width>105</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Directory: </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>76</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="overwriteMode">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="catalogTiffBrowseButton_2">
      <property name="geometry">
       <rect>
        <x>652</x>
        <y>72</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="catalogTiffOutputWindow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>192</y>
        <width>755</width>
        <height>325</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>124</y>
        <width>153</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Fan out by resolution</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>158</y>
        <width>745</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="copyTiffs">
     <attribute name="title">
      <string>Copy Tiffs</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>32</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Geotiff Catalog Directory:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>64</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Intersecting Shapepfile: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>96</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Geotiff Output Directory: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>32</y>
        <width>433</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>64</y>
        <width>433</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>96</y>
        <width>433</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>28</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>92</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>124</y>
        <width>153</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Fan out by resolution</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="catalogTiffOutputWindow_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>192</y>
        <width>755</width>
        <height>325</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>158</y>
        <width>745</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="qkShp">
     <attribute name="title">
      <string>Shapefile QuickView</string>
     </attribute>
     <widget class="QGraphicsView" name="shapeViewWindow">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>54</y>
        <width>763</width>
        <height>433</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>20</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Shapefile: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="shapefileViewPath">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>18</y>
        <width>433</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="shapefileViewBrowseButton">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>14</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shapefileViewClear">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shapefileViewGo">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>490</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Shape</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
